src/lua/debug.lua

```diff
@@ -0,0 +1,28 @@
+local debug = require('debug')
+
+local function sourcepath(level)
```

Comment:
We discussed it shortly with Vladimir and found that `sourcedir()` looks consistent with `__dir__`, while `sourcepath()` doesn't look consistent with `__file__`. Maybe it worth to rename it to `sourcefile()`?

Reply:
Renamed `sourcepath` to `sourcefile` ✔️